client/modules/User/components/Collection.jsx

```diff
@@ -106,7 +108,7 @@ const CollectionItemRowBase = ({
           <button
             className="collection-row__remove-button"
             onClick={handleSketchRemove}
-            aria-label="Remove sketch from collection"
+            aria-label={t('Collection.SketchDeletedSpan')}
```

Comment:
`Collection.SketchDeletedSpan` is the same translation key as L89 above. It should be different?

Reply:
Mistake, corrected using the proper ARIA label